app/src/main/java/com/example/instagramclone/fragments/message_screen/compose/MessageItem.kt

```diff
@@ -0,0 +1,75 @@
+package com.example.instagramclone.fragments.message_screen.compose
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import com.example.domain.entities.Message
+import com.example.instagramclone.R.drawable
+
+@Composable
+fun MessageItem(messageItem: Message) {
+    Row(
+        modifier = Modifier
+            .height(74.dp)
+            .padding(horizontal = 12.dp)
+            .fillMaxWidth(),
+        horizontalArrangement = Arrangement.SpaceBetween,
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        Row() {
+            Box(
+                contentAlignment = Alignment.Center
+            ) {
+                Image(
+                    modifier = Modifier
+                        .size(64.dp),
+                    painter = painterResource(id = drawable.ic_story_border_rainbow),
+                    contentDescription = "story border"
+                )
+                AsyncImage(
+                    modifier = Modifier
+                        .size(54.dp)
+                        .clip(CircleShape),
+                    contentScale = ContentScale.Crop,
+                    placeholder = painterResource(id = drawable.bg_placeholder),
+                    model = "https://picsum.photos/id/${messageItem.id + 10}/500/500",
+                    contentDescription = "profile image"
+                )
+            }
+            Column(
+                modifier = Modifier
+                    .fillMaxHeight()
+                    .padding(horizontal = 12.dp),
```

Comment:
It's best to use the [4-point grid system](https://medium.com/@aratidube12lns/4-point-grid-system-for-more-consistent-interface-design-efea81dea3f3#:~:text=What%20is%20the%204%2Dpoint,padding%2C%20margins%20and%20line%20heights.) wherever possible.

And let's extract all string resources in this file